.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "data-plane-api"]
-	path = data-plane-api
-	url = https://github.com/envoyproxy/data-plane-api
```

Comment:
?? why remove this?

Reply:
The stubs will be pushed from envoy repo directly. I want to have the first commit done manually before I automate the pushes since we still need to validate go module build (which bazel doesn't do).

Reply:
See https://github.com/envoyproxy/envoy/pull/8155